src/components/HTMLEngineProvider/HTMLRenderers/ConciergeLinkRenderer.tsx

```diff
@@ -19,11 +19,15 @@ function navigateToConciergeChat() {
 function ConciergeLinkRenderer({tnode}: ConciergeLinkRendererProps) {
     const styles = useThemeStyles();
 
+    const isChildOfMutedTextLabel = tnode.parent?.domNode?.name === 'muted-text-label';
     // Define link style based on context
     let linkStyle: StyleProp<TextStyle> = styles.link;
 
+    if (isChildOfMutedTextLabel) {
+        linkStyle = [styles.link, {fontSize: styles.mutedNormalTextLabel.fontSize}];
+    }
```

Comment:
We already wrapped the text with `<muted-text-label>`, so in theory, we won't need to conditionally apply this style anymore. We can take the `style` prop and pass it to the `Text` style below.

Reply:
> you want to add a new prop style and provide the values from the parent?

We won't provide the value. It will be provided by the renderer. It's like the parent style. So, in our case,
`<muted-text-label> ... <concierge-link>Concierge</concierge-link>...</muted-text-label>`

the concierge link will get the style from `<muted-text-label>`, and we will append it with `linkStyle`.

```
style={[style, linkStyle]}
```

> wouldnt that be a breaking change for all existing ConciergeLinkRenderer ?

I think there is always a possibility of breaking change, but it should be the way of handling a custom renderer and cascading the style down to children. So, the current code is more like a workaround, and I'm afraid in the future, more workarounds will be added.